docs/ref/django-admin.txt

```diff
@@ -412,6 +412,10 @@ Specifies the database to introspect. Defaults to ``default``.
 
 If this option is provided, models are also created for database views.
 
+.. versionchanged:: 2.2
```

Comment:
This note needs to be self-contained. Something like:
```
On PostgreSQL, models are also created for materialized views.

.. versionchanged:: 2.2

    Older versions don't create tables for materialized views.
```

Reply:
Ok.